### PR TITLE
Update: login-signup pages

### PR DESCRIPTION
**Pull request by bossmonkey98**
Login Sign up Page Updates
Document — Mozilla Firefox 16-02-2022 03_28_31
Document — Mozilla Firefox 16-02-2022 03_28_39

### CODE REVIEW

File: components/pages/aunthentication/auth.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+.btn{
+    font-size: large;
+    background-color: var(--primary-color);
+    margin: 1.5rem;
+    width: 8rem;
+    color: black;
+}
+
+.sign-up{
+    color:tomato;
```
Comment (t007rushi) [inline components/pages/aunthentication/auth.css:41]: Use color variable

**Review by t007rushi — APPROVED** [review]
Everything looks clean

**Review by sanjitsarkar — APPROVED** [review]
Everything looks good except not using color variable.

File: components/pages/aunthentication/auth.css (hunk 1 above)
Comment (bossmonkey98) [reply components/pages/aunthentication/auth.css:41]: Is it necessary to use variable if I'm going to use the color only once?

**Pull request merged**